src/main.c

```diff
@@ -93,6 +93,8 @@ int main(int argc, char *argv[])
         return throw_err("Failed to load initrd");
     if (diskimg_file && vm_load_diskimg(&vm, diskimg_file) < 0)
         return throw_err("Failed to load disk image");
+    if (vm_enable_net(&vm) < 0)
```

Comment:
If virtio-net is not enabled, it still makes sense to initialize and launch this virtual machine via KVM. Thus, instead of returning, we can proceed.

Reply:
I changed throw_err to fprintf. If vm_enable_net encounters a failure, it will now handle the situation by logging an error message.
```c
if (vm_enable_net(&vm) < 0)
        fprintf(stderr, "Failed to Enable Virtio-Net Device\n");
```

Reply:
Refine the message as following:
> Failed to enable virtio-net device.

Reply:
Ok ! Already done